examples/all-clusters-app/linux/include/CHIPProjectAppConfig.h

```diff
@@ -47,3 +47,7 @@
 #define CHIP_CONFIG_MAX_PATHS_PER_INVOKE 5
 
 #define CHIP_CONFIG_ENABLE_BDX_LOG_TRANSFER 1
+
+// Claim a device type while advertising that matches the device type on our
```

Comment:
should we say that 0x0101 is "dimmable light" ?

Reply:
Yeah, I can do that.